app/models/v2/test_result.rb

```diff
@@ -10,7 +10,10 @@ class TestResult < ApplicationRecord
     belongs_to :system, class_name: 'V2::System', optional: true
     belongs_to :tailoring, class_name: 'V2::Tailoring'
 
+    has_one :profile, class_name: 'V2::Profile', through: :tailoring
+    has_one :security_guide, class_name: 'V2::SecurityGuide', through: :profile
     has_one :policy, class_name: 'V2::Policy', through: :tailoring
-    has_one :security_guide, class_name: 'V2::SecurityGuide', through: :tailoring
+    has_one :report, class_name: 'V2::Report', through: :policy
```

Comment:
Why is this not `belongs_to`?

Reply:
`belongs_to` requires a field pointing to that record, indirect relations can be only achieved by `through`